src/csharp/SDK/Capture.cs

```diff
@@ -169,98 +157,235 @@ public Image Color
                         throw new ObjectDisposedException(nameof(Capture));
                     }
 
-                    // If the assignment is a new managed wrapper we need
-                    // to release the reference to the old wrapper
-                    // If it is the same object though, we should not dispose it.
-                    if (this._IR != null && !object.ReferenceEquals(
-                        this._IR, value))
-                    {
-                        this._IR.Dispose();
-                    }
-                    this._IR = value;
+                    NativeMethods.k4a_capture_set_temperature_c(this.handle, value);
                 }
             }
         }
 
-        public float Temperature
+        /// <summary>
+        /// Creates a new capture.
+        /// </summary>
+        /// <returns>A new Capture instance.</returns>
+        public static Capture Create()
```

Comment:
What is the benefit of this class since we already have a public constructor?

Reply:
I agree.